pkg/reconciler/kubernetes/adapter/adapter.go

```diff
@@ -200,8 +200,11 @@ func (g *kubeClientAdapter) Delete(ctx context.Context, manifest, namespace stri
 		g.logger.Warnf("Watching progress of deleted resources failed: %s", err)
 	}
 
-	//TODO: check if namespace is empty and if yes: remove it
-
+	if err = g.kubeClient.DeleteNamespace(namespace); err != nil {
+		g.logger.Errorf("Failed to delete namespace name='%s': %s",
```

Comment:
This is now deleting the namespace without checking for custom resources, right?

Reply:
in first version we check if namespace contains any resources. If any resources exist on namespace, we don't remove namespace.